README.md

```diff
@@ -14,15 +31,19 @@ While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great
 Swift-Kuery is easy to learn, consumable framework that comes with a set of [implemented plugins](#list-of-plugins).
 
 ## Table of Contents
-* [Example](#example)
+* [Examples](#examples)
 * [SQL Injection Prevention using Parameterization](#sql-injection-prevention-using-parameterization)
 * [Prepared Statements](#prepared-statements)
 * [Schema Management](#schema-management)
 * [Query Examples](#query-examples)
 * [List of plugins](#list-of-plugins)
 * [License](#license)
 
-## Example
+## Examples
+A Swift-Kuery tutorial on adding a PostgreSQL database to a foodTracker app is available [here](https://github.com/IBM/FoodTrackerBackend/blob/master/AddDatabase.md).
+
+An example ToDo list application, implementing Swift-Kuery, is available [here](https://github.com/IBM-Swift/iOSSampleKituraKit/tree/persistentiOSKituraKit).
+
 This example demonstrates how to execute an SQL query using Swift-Kuery and Swift-Kuery-PostgreSQL. It assumes there is a PostgreSQL server running on localhost:5432, that contains a table called Grades:
```

Comment:
Line 64 - remove 'we' from "First we import Swift-Kuery and Swift-Kuery-PostgreSQL:"

Line 71 - reword to "Now create a class of type `Table` that corresponds to our Grades table in PostgreSQL:"

Line 83 - remove 'we' - "Next we create a pool of connections to the PostgreSQL database:"

Change the `options: [.userName("username"), .password("password")]` in the following code to `options: [.databaseName("school")]`
```swift
 let pool = PostgreSQLConnection.createPool(host: "localhost", port: 5432, options: [.userName("username"), .password("password")], poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50, timeout: 10000))
```